internal/nodes/avi_model_l7_translator.go

```diff
@@ -32,19 +32,6 @@ import (
 // TODO: Move to utils
 const tlsCert = "tls.crt"
 
-func RemoveFQDNsFromModel(vsNode *AviVsNode, hosts []string, key string) {
```

Comment:
The removal logic was not working if the hosts contains more than one value.

Refer here for a [sample program](https://go.dev/play/p/yAZGDW5ADHA) illustrating it.